internal/channel_tags/routes.go

```diff
@@ -0,0 +1,112 @@
+package channel_tags
+
+import (
+	"database/sql"
+	"fmt"
+	"github.com/gin-gonic/gin"
+	"github.com/jmoiron/sqlx"
+	"github.com/lncapital/torq/internal/corridors"
+	"github.com/lncapital/torq/pkg/server_errors"
+	"github.com/rs/zerolog/log"
+	"gopkg.in/guregu/null.v4"
+	"net/http"
+	"strconv"
+)
+
+func RegisterChannelTagRoutes(r *gin.RouterGroup, db *sqlx.DB) {
+	r.POST("", func(c *gin.Context) { addChannelTagHandler(c, db) })
+	r.DELETE(":channelTagId", func(c *gin.Context) { removeChannelTagHandler(c, db) })
+}
+
+func addChannelTagHandler(c *gin.Context, db *sqlx.DB) {
+	var ct channelTag
+	if err := c.BindJSON(&ct); err != nil {
+		server_errors.SendBadRequestFromError(c, err)
```

Comment:
nit: could have wrapped error with a bit of context just incase the error out of BindJSON is not clear.
```suggestion
		server_errors.SendBadRequestFromError(c, errors.Wrap(err, "Parsing JSON"))
```